app/controllers/sections_controller.rb

```diff
@@ -1,10 +1,44 @@
 class SectionsController < ApplicationController
 
   def index
+    all_videos = Video.order(:created_at)
+    @latest_videos = []
+    unless all_videos.empty?
+      @latest_videos = [
+        all_videos.last,
+        all_videos.second_to_last,
+        all_videos.third_to_last
+      ]
+    end
   end
 
   def show
     @section = Section.find(params[:id])
-    @latest_videos = [Video.last, Video.second_to_last, Video.third_to_last]
+    @subsections = @section.subsections
+    @latest_videos = []
+
+    if @section.name == 'Eventos & Pesquisa'
+      subsection_events = @subsections.find_by(name: 'events')
+      highlight_category = subsection_events.categories.find_by(name: 'Em destaque')
```

Comment:
Use `highlighted` column from `videos` and/or `series` table instead of a custom category for highlight.